GiniVision.podspec

```diff
@@ -15,11 +15,24 @@ The Gini Vision Library for iOS provides functionality to capture documents with
   s.author           = { 'Peter Pult' => 'p.pult@gini.net' }
   s.source           = { :git => 'https://github.com/gini/gini-vision-lib-ios.git', :tag => s.version.to_s }
   s.social_media_url = 'https://twitter.com/gini'
-
   s.ios.deployment_target = '8.0'
+  s.default_subspec = 'Core'
+
+  s.subspec 'Core' do |core|
+    core.source_files = 'GiniVision/Classes/Core/**/*'
+    core.resources = 'GiniVision/Assets/*'
+  end
+
+  s.subspec 'Networking' do |networking|
+    networking.source_files = 'GiniVision/Classes/Networking/*.swift', 'GiniVision/Classes/Networking/Extensions/*.swift'
+    networking.dependency "Gini-iOS-SDK", "~> 0.6.0"
+  end
 
-  s.source_files = 'GiniVision/Classes/**/*'
-  s.resources = 'GiniVision/Assets/*'
+  s.subspec 'Networking+Pinning' do |pinning|
+    pinning.source_files = 'GiniVision/Classes/Networking/Pinning/*', 'GiniVision/Classes/Networking/*.swift', 'GiniVision/Classes/Networking/Extensions/*'
+    pinning.dependency "Gini-iOS-SDK", "~> 0.6.0"
+    pinning.dependency "Gini-iOS-SDK/Pinning", "~> 0.6.0"
```

Comment:
This version (0.6.0) is repeated many times. And I assume it HAS to always be the same. Can you use a variable to make it less error prone? like `s.version` above.

Reply:
I cannot. At least I didn't find a way to add a custom parameter in the Podspec.